Configurations/windows-makefile.tmpl

```diff
@@ -762,8 +775,8 @@ EOF
           # Also redo $gen0, to ensure that we have the proper extension.
           $gen0 = platform->bin($gen0);
           return <<"EOF";
-$args{src}: $gen0 $deps
-	$gen0$gen_args > \$@
+$args{src}: $gen0 $deps "\$(BLDDIR)/util/wrap.pl"
```

Comment:
Should backslashes be used here? But they would have to be probably doubled?

Reply:
I thought of using backslashes, but since it's not an ordinary make rule but part of the Perl code I didn't dare to do it because I don't really know how many levels of quoting it would require inside the string (perl, make, cmd, ...). But I can give it another try.

I also noticed that the Windows makefile is terribly inconsistent about pathes need to be quoted and which not (e.g., those containing `$(SRCDIR)` or `$(BLDDIR)`).

https://github.com/openssl/openssl/blob/60caee4bf1c780e95e2853b785e94ef65c8f2621/Configurations/windows-makefile.tmpl#L610-L614

Reply:
Converted to backslashes in 55e52aabd1aa27d1749eb099c0803d61bb49b06c.